sdk/spring/azure-spring-boot-starter-cosmos/README.md

```diff
@@ -47,19 +47,18 @@
 Open `application.properties` file and add below properties with your Cosmos DB credentials.
 
 ```properties
-azure.cosmosdb.uri=your-cosmosdb-uri
-azure.cosmosdb.key=your-cosmosdb-key
-azure.cosmosdb.database=your-cosmosdb-databasename
+azure.cosmos.uri=your-cosmos-uri
+azure.cosmos.key=your-cosmos-key
+azure.cosmos.database=your-cosmos-database-name
 ```
 
-Property `azure.cosmosdb.consistency-level` is also supported.
+Property `azure.cosmos.consistency-level` is also supported.
 
-Property `azure.cosmosdb.cosmosKeyCredential` is also supported. CosmosKeyCredential feature provides capability to 
-rotate keys on the fly. You can switch keys using switchToSecondaryKey(). For more information on this, see the Sample 
-Application code.
+Property `azure.cosmos.credential` is also supported. AzureKeyCredential feature provides capability to 
```

Comment:
We should make sure the key rotation is really supported now.

Reply:
I tested the 2 cases, it's ok to configure only `azure.cosmos.key` or `azure.cosmos.credential`, the rotation feature should be supported now, the [sample ](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/SampleApplication.java)will be found in spring cosmos module.